generate.sh

```diff
@@ -312,7 +386,7 @@ cat << EOF >> ${DST_KEYS}
 // ${SRC_CONST}
 EOF
 
-read_enum iproto_raft_keys ${SRC_CONST} | \
+read_define_with_prefix IPROTO_RAFT_KEYS iproto_raft_key ${SRC_CONST} | \
 	generate_enum RaftKey >> ${DST_KEYS}
```

Comment:
Please, fix formatting in the file:

Align with `\t` or `  ` in the whole file.

Reply:
Updated.